tracer/test/snapshots/Iast.TrustBoundaryViolation.AspNetMvc5.IastEnabled.verified.txt

```diff
@@ -0,0 +1,125 @@
+﻿[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: aspnet.request,
+    Resource: GET /iast/tbv,
+    Service: sample,
+    Type: web,
+    Tags: {
+      env: integration_tests,
+      http.method: GET,
+      http.request.headers.host: localhost:00000,
+      http.route: {controller}/{action}/{id},
+      http.status_code: 200,
+      http.url: http://localhost:00000/Iast/Tbv?name=name&value=value,
+      http.useragent: Mistake Not...,
+      language: dotnet,
+      runtime-id: Guid_1,
+      span.kind: server,
+      _dd.iast.enabled: 1,
+      _dd.iast.json:
+{
+  "vulnerabilities": [
+    {
+      "type": "TRUST_BOUNDARY_VIOLATION",
+      "evidence": {
+        "valueParts": [
+          {
+            "value": "name",
```

Comment:
We don't give the type / method name where the issue occurs?

Reply:
Yes, in the location. But that part is scrubbed